### ISSUE TITLE
adx_table_streaming_ingestion_policy.hint_allocated_rate strconv.ParseFloat: parsing "": invalid syntax

### ISSUE THREAD

**Issue by guy-radford-sunswap** [favoretti/terraform-provider-adx#30]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Terraform Version

v1.3.7

### ADX Provider Version

0.0.18

### Terraform Configuration Files

```hcl
resource "adx_table_streaming_ingestion_policy" "iothub_received_raw_events_streaming_ingestion" {
  database_name       = local.axd_database_name
  table_name          = adx_table.iothub_received_raw_events.name
  enabled             = true
  hint_allocated_rate = 2.1
}

--- OR ----

resource "adx_table_streaming_ingestion_policy" "iothub_received_raw_events_streaming_ingestion" {
  database_name       = local.axd_database_name
  table_name          = adx_table.iothub_received_raw_events.name
  enabled             = true
}
```

### Debug Output/Panic Output

```shell
╷
│ Error: error parsing hint_allocated_rate from ADX API for policy streamingingestion for Table "iothub_received_raw_events" (Database "adxdb-endurance-dev-uksouth"): strconv.ParseFloat: parsing "": invalid syntax
│ 
│ 
╵
```

### References

https://registry.terraform.io/providers/favoretti/adx/latest/docs/resources/adx_table_streaming_ingestion_policy

I have tried both with `hint_allocated_rate` set and commented out.

I had the streaming policy already enabled, so I removed it and tried again, but that didnt work either.

I was intending on using the default value for `hint_allocated_rate`.

I will try to destroy everything and retry, after I have finished the other changes i am making.

**Comment by slwbuild** [comment]
Thanks for reporting this, obviously my acceptance test for this feature was a bit weak. 

I'll get this reproduced and resolved.

**Comment by guy-radford-sunswap** [comment]
I can confirm this happens when creating everything from scratch too. 
I tried adding a `depends_on` referencing back to the `table`, but to no avail.

Thank you

**Comment by slwbuild** [comment]
We will likely release this fix later today v0.0.19
